### DISCUSSION TITLE
Custom Links, in the same window

### DISCUSSION THREAD

**Discussion by TLMondo** [directus/directus#20297] [Feature Requests]
### Summary

Allow for `Custom Links` in the `Module Bar` to be set to either `Current Tab` or `New Tab`.

<img width="1082" alt="image" src="https://github.com/directus/directus/assets/7502754/ad09a78d-b7c4-4217-a1ed-b657440c07d2">

### Basic Example

I've created a link to the `Data Model` section under Settings, but it opens a new tab.

### Motivation

There are times we want shortcut users to specific area's of the admin ui, like `Data Model`, because they are frequently confused as to where it lives. It's a little unintuitive for a new user to think to go to the Setting section to then manage the model.

### Detailed Design

N/A

### Requirements List

N/A

### Drawbacks

None?

### Alternatives

Create a stand-alone built-in module for `Data Model`

### Adoption Strategy

Just add a drop-down, and default it to "New Tab" so that its easy to transition.

**Comment by rijkvanzanten** [comment]
Makes a ton of sense to me. I see it is a tiny improvement on an existing feature, so lets move it to an issue ready to be implemented 🙂 

Thanks for the suggestion!

**Reply by Seinnd** [reply]
I'm pretty sure that internal links are already opened internally 🤔 @TLMondo 

What I mean to say is that the desired behavior already exists if you stick to common URL request patterns.
![image](https://github.com/directus/directus/assets/22862290/762309a5-f04d-4280-b5d1-30337d9421b3)
![image](https://github.com/directus/directus/assets/22862290/4564c1b2-436b-4fb0-91bf-61fc9b8d1f7c)

For clarification, even if this seems only marginal, but if the users are given the option to have external addresses redirected to the current window, a completely new request is created - that is not in the sense of the inventor of client-side outsourced user interfaces.

**Reply by rijkvanzanten** [reply]
Ohhhhh right I totally forgot! Thanks for the reminder @Seinnd 🙂

**Reply by TLMondo** [reply]
Great. I was using full url (probably out of laziness), and saw other behaviour.
I've switched to `/settings/data-model` and it works.  `/admin/settings/data-model` brings you to --> `/admin/admin/settings/...`

**Comment by rijkvanzanten** [comment]
This feature request has been accepted and is queued to be implemented! You can follow along with the progress here:  https://github.com/directus/directus/issues/20298

**Reply by rijkvanzanten** [reply]
https://github.com/directus/directus/issues/20298#issuecomment-1790817578!
